marklogic-data-hub-central/ui/src/components/entities/mapping/entity-map-table/entity-map-table.tsx

```diff
@@ -993,14 +994,15 @@ const EntityMapTable: React.FC<Props> = (props) => {
     return (
       <Dropdown overlay={functionMenu} trigger={["click"]} disabled={!props.canReadWrite}>
         <Tooltip title={props.canReadWrite && "Function"} placement="bottom">
-          <Button
+          <HCButton
```

Comment:
I'm seeing the function buttons broken in mapping, they need a CSS width fix?
<img width="415" alt="function-buttons" src="https://user-images.githubusercontent.com/477757/132906648-66e403bd-de49-4843-a801-3026bff87703.png">

Reply:
@wooldridge @jbelonoj this is the looks for the function buttom
![image](https://user-images.githubusercontent.com/88790827/133125103-6226c1e3-bfd2-40d0-8f91-eb5e3b6fc9e6.png)

Reply:
Thew function buttons should have a white background, currently they are transparent. Here's what they look like on the develop nightly:
<img width="131" alt="function-buttons2" src="https://user-images.githubusercontent.com/477757/133133725-aa579020-ee38-4baf-a767-8f7d4a480cf3.png">

Reply:
@wooldridge this was solved too 
![image](https://user-images.githubusercontent.com/88790827/133154101-cfff9794-8a6e-49b6-a321-24c4c05203a2.png)